### DISCUSSION TITLE
How does salting enhance the security of hashed passwords, and what potential risks are mitigated by using salts?

### DISCUSSION THREAD

**Discussion by anhducokok** [Yuukicoder/Gas_Store-#51] [Q&A] [answered]
How does salting enhance the security of hashed passwords, and what potential risks are mitigated by using salts?

**Comment by Yuukicoder** [comment, marked as answer]
Salting enhances the security of hashed passwords primarily by adding a unique random value to each password before hashing it. This random value, known as a salt, ensures that even if two users have the same password, their hashed passwords will be different due to the unique salt applied to each one.
